db/disk/submission.go

```diff
@@ -323,3 +323,32 @@ func (this *backend) RemoveSubmission(assignment *model.Assignment, email string
 
     return true, nil;
 }
+
+func (this *backend) GetAttempts(assignment *model.Assignment, email string) ([]*model.GradingResult, error) {
```

Comment:
Let's change this function name to "GetSubmissionAttempts" to mirror "GetSubmissionHistory", etc.